contracts/staking/package.json

```diff
@@ -51,6 +51,8 @@
     "devDependencies": {
         "@0x/abi-gen": "^4.1.0",
         "@0x/contracts-gen": "^1.0.13",
+        "@0x/contracts-exchange": "^2.1.14",
```

Comment:
I may end up making an `integration-tests` package eventually because I'd really like to add the forwarder, coordinator, etc. to this test. I'll just move to the exchange for now though.